examples/Example/src/prot_drag/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useRef } from 'react';
+import { StyleSheet, View, Animated } from 'react-native';
+import {
+  GestureMonitor,
+  useGesture,
+  Gesture,
+} from 'react-native-gesture-handler';
+
+function getState(s: number) {
+  switch (s) {
+    case 0:
+      return 'Undetermined';
+    case 1:
+      return 'Failed';
+    case 2:
+      return 'Began';
+    case 3:
+      return 'Cancelled';
+    case 4:
+      return 'Active';
+    case 5:
+      return 'End';
+  }
+  return s;
+}
+
+function Draggable() {
+  const translationX = useRef(new Animated.Value(0)).current;
+  const translationY = useRef(new Animated.Value(0)).current;
+  const scale = useRef(new Animated.Value(1)).current;
```

Comment:
`useMemo(() => new Animated.Value(0), [])` would be better here.
Right now you're creating a new animated value every time this function is called (so, on every render) and except the first time, those animated values are immediately GCed (since nothing references them).

Reply:
I converted this example to `Reanimated` in 920c25d1a5b20a6da43a02e53d0d1b4e8243a894, because `GestureMonitor` is not working with native `Animated` anyway